internal/observability/middleware_test.go

```diff
@@ -268,7 +268,12 @@ func TestNewMiddlewareLogLevels(t *testing.T) {
 }
 
 func TestMiddlewareLoggingWithApplicationErrorConfiguration(t *testing.T) {
-	defer stubTime()()
+	timeVal := time.Now()
+	defer stubTimeWithTimeVal(timeVal)()
+	ttlMs := int64(1000)
```

Comment:
nit: set `ttlMs` to `time.Millisecond * 1000` and use it directly in below cases. (also applies to TestUnaryInboundApplicationErrors and TestMiddlewareLoggingWithServerErrorConfiguration)

Reply:
done